lib/slack/block_kit/composition/conversation_filter.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Slack
+  module BlockKit
+    module Composition
+      # Provides a way to filter the list of options in a conversations select menu
+      # or conversations multi-select menu.
+      #
+      # @param [Array] conversation_types - "include" field
+      #
+      # https://api.slack.com/reference/block-kit/composition-objects#filter_conversations
+      # https://api.slack.com/reference/block-kit/block-elements#conversation_multi_select
+      # https://api.slack.com/reference/block-kit/block-elements#conversation_select
+      class ConversationFilter
+        def initialize(conversation_types: nil,
```

Comment:
What do you think of using `only` or `includes`?

Reply:
I don't like `includes` because it's look like a typo when compare to API's document.
`only` is a bit too wide as it may contain more context than just conversation types. But it's good to me, we could make it clear by document and validation later. 

Look still nice 😄 

```ruby
select = ConversationsSelect.new
select.filter(only: ['im', 'public'])
```

You ok with `only`? @CGA1123

Reply:
I think `only` makes sense, another alternative could be `include_` which looks a bit awkward but is a common pattern to avoid clashes with keywords in other languages